has_many_blogs.sql

```diff
@@ -0,0 +1,39 @@
+-- \c postgres
+-- DROP DATABASE IF EXISTS has_many_blogs;
+
+-- CREATE USER has_many_posts;
+
+
+-- CREATE DATABASE has_many_blogs WITH OWNER has_many_posts;
+-- \c has_many_blogs
+
```

Comment:
When dropping tables you can not drop a table that is being referred to. In this case `users` is being referenced by `posts` and `comments`.  A general rule is to `DROP` in the reverse order the tables were created.